katacomb/katacomb/mock_dataset.py

```diff
@@ -450,16 +450,16 @@ def _generate_ref_ant_compound_scans():
                 dump_index += dumps
 
         # Generate compound scans for the reference antenna
-        ref_ant_compound_scans = list(_generate_ref_ant_compound_scans())
+        ref_ant_compound_scans = _generate_ref_ant_compound_scans()
```

Comment:
Is this `list` safe to remove?

Reply:
Yes- I tested it. It's just returning a generator object that is looped over on the next line in a list comprehension.